crates/zed/src/zed.rs

```diff
@@ -210,6 +210,12 @@ pub fn initialize_workspace(
     .detach();
 
     feature_gate_zed_pro_actions(cx);
+    if cx.can_select_mixed_files_and_dirs() {
```

Comment:
Do we need to filter this out? I think this kind of effective redundancy is fine for letting people's knowledge of Zed be portable across platforms :)

Reply:
Fair enough!